src/subdomains/core/sell-crypto/process/dto/update-buy-fiat.dto.ts

```diff
@@ -197,4 +197,12 @@ export class UpdateBuyFiatDto {
   @IsDate()
   @Type(() => Date)
   chargebackAllowedDate: Date;
+
+  @IsOptional()
+  @IsInt()
+  bankDataId: number;
+
+  @IsOptional()
+  @IsBoolean()
+  activateBankData: boolean;
```

Comment:
Maybe `bankDataActive` is a better name? Because `activateBankData=false` sounds like nothing is done, but it will deactivate the bank data.
Same for buy-crypto.